test/fuzz/transformation_add_parameter_test.cpp

```diff
@@ -164,6 +168,15 @@ TEST(TransformationAddParameterTest, NonPointerBasicTest) {
         context.get(), validator_options, kConsoleMessageConsumer));
     ASSERT_TRUE(transformation_context.GetFactManager()->IdIsIrrelevant(66));
   }
+  // Adds array as parameter to a function.
+  {
+    TransformationAddParameter correct(9, 68, 52, {{{13, 54}}}, 69);
```

Comment:
If you look at the TransformationAddParameter message in spvtoolsfuzz.proto, you'll see that the second argument, 68 in your case - is the id of the new parameter.

As a result, the additional OpFunctionParameter instruction below, which you've given result id %70, should have result id %68.

That explains one of the discrepancies in the test results.

Reply:
Done